public/js/saveApplicant.js

```diff
@@ -15,19 +21,16 @@ document.getElementById('submitApplicant').addEventListener('click', e => {
                 document.querySelector(querySelector).classList.remove('hidden');
                 document.querySelector(querySelector).innerHTML = errorMessage(validationType);
                 formIsValid = false;
-            } else {
-                document.querySelector(querySelector).classList.remove('alert-danger');
-                document.querySelector(querySelector).classList.add('hidden');
-                document.querySelector(querySelector).innerHTML = '';
```

Comment:
this also could be a bit more DRY but tbh it reads well as well so Its up to you if you want to change it. I am happy either way.